ui/DesignSystem/Component/PeerSelector/Peer.svelte

```diff
@@ -33,5 +33,9 @@
     <p class="badge">
       <Badge variant={BadgeType.Maintainer} />
     </p>
+  {:else if peer.type === PeerType.Local}
+    <p class="badge">
+      <Badge variant={BadgeType.You} />
+    </p>
```

Comment:
Instead of having to have these `if` checks every time we want to display the `Badge` and have the `Badge` receiving a variant, we could have `Badge` receiving the `peer.type` value and have this conditional check unified in `Badge`.

Reply:
Don't know if that'll work always, cause sometimes we use that to actually decide to show the badge or not. That logic wouldn't make sense to have in the component, right?